website/docs/reference/header-contract.md

```
@@ -0,0 +1,126 @@
+await js.publish(
+    subject="orders__microservice.ev.orders.created",
+    payload=payload,
+    headers=headers,
+)
+```
+
+### Reading the trace context manually
+
+If you cannot use OpenTelemetry on the consumer side, you can still inspect the W3C trace headers yourself. The `traceparent` header has the form:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)